recipes/spacepy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+package:
+  name: spacepy
+  version: "0.1.6"
+
+source:
+  fn: spacepy-0.1.6.tar.gz
+  url: https://pypi.python.org/packages/f7/10/4a9a43014d25c1ce08aad136da2515eb120a26105dec5e8f42b5986c22a3/spacepy-0.1.6.tar.gz
+  md5: 1272ec6537ca45b6e4b8ea3f8c2cf8a1
+
+build:
+  number: 0
+  features:
+    - vc9      [win and py27]
+    - vc10     [win and py34]
+    - vc14     [win and py35]
+
+requirements:
+  build:
+    - python
+    - numpy
```

Comment:
In principle we need `numpy.distutils` for the build step, how do we know if we need to link against NumPy?